docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -1,40 +1,47 @@
 
 = OpenNMS Time Series DB
 
-The OpenNMS Time Series DB is a cloud-hosted service that provides a scalable time series database to store and persist performance metrics that {page-component-title} collects from the devices that it monitors.
+The OpenNMS Time Series DB is a cloud-hosted service that provides a scalable time series database to store and persist performance metrics that {page-component-title} collects from monitored devices.
 
 == Requirements
 
 You must have the following to use the Time Series DB:
 
 * Horizon 31+ (Meridian availability with the 2023 release).
-* A Time Series DB subscription, which provides access to the https://portal.opennms.com[OpenNMS Portal] to retrieve your activation key and view Time Series DB metrics.
-Contact sales@opennms.com to sign up for the Time Series DB.
+* A Time Series DB subscription.
+The subscription provides access to the https://portal.opennms.com[OpenNMS Portal], where you can retrieve your activation key and view Time Series DB metrics.
+Contact sales@opennms.com to sign up.
+
+NOTE: When you sign up for the Time Series DB subscription, you receive an activation email from The OpenNMS Group with a login for the OpenNMS Portal.
+If you do not receive the activation email, contact your account manager.
 
 == Deployment
 
-There are four steps to set up the Time Series DB for your system:
+Follow these steps to set up the Time Series DB in your environment:
 
-. Update OpenNMS Cloud Services Connector files.
-. Configure plugin settings.
+. Update your OpenNMS Cloud Services Connector files.
+. Configure Cloud Services settings.
 . Obtain an activation key for the Cloud Services Connector.
-. Enable cloud services connectivity.
+. Enable Cloud Services connectivity.
+. Allow {page-component-title} to make outgoing connections to the following gateways:
```

Comment:
This should be checked first, as you can't enable the cloud services unless the firewall is allowing access to these hosts.
We may also need to provide IP addresses, as firewalls typically base their rules on IP and not the DNS name.

Reply:
Would it be sufficient to include an admonition for the firewall check in the step's section, or are there specific steps we should write out to guide the user through checking this?